cmd/query/app/querysvc/adjuster/adjuster.go

```diff
@@ -9,20 +9,12 @@ import (
 	"go.opentelemetry.io/collector/pdata/ptrace"
 )
 
-// Adjuster defines an interface for modifying a trace object.
-// It returns the adjusted trace object, which is also updated in place.
+// Adjuster defines an interface for modifying a trace object in place.
 // If the adjuster encounters an issue that prevents it from applying
-// modifications, it should return the original trace object along with an error.
+// modifications, it should return an error. The original trace object
+// is modified directly and not returned.
```

Comment:
I would add that the caller must guarantee that all spans in `ptrace.Traces` argument must belong to the same trace and represent the complete trace.